src/backend/gl/src/info.rs

```diff
@@ -566,7 +744,20 @@ pub(crate) fn query_all(
         memory_barrier: info.is_supported(&[Core(4, 2), Es(3, 1)]),
     };
 
-    (info, features, legacy, properties, private)
+    let filter = if info.is_supported(&[Es(3, 0)]) {
+        /* Use the OpenGL ES 3 format filter. */
+        TextureFormatFilter::new_es3()
+    } else if info.is_supported(&[Es(1, 0)]) {
+        /* Use the OpenGL ES 1 and OpenGL ES 2 format filter. */
+        TextureFormatFilter::new_es1_es2()
+    } else {
+        /* We're using the core specification. We can assume all of the
+         * combinations are valid, provided the OpenGL enums values are also
+         * valid for textures. */
+        TextureFormatFilter::new()
```

Comment:
this is an empty list, isn't it? So for the core spec we'd end up not exposing any formats at all, which is not the intent here.

Reply:
Yeah, I got that wrong

Reply:
Corrected it in [5c88051](https://github.com/gfx-rs/gfx/pull/3686/commits/5c880518d7acbb6c8b2acb726fdba46ec3315b72).